Merge problem-definition-40 into develop (#45)
SW709E19/Report: report/PartProblemAnalysis/ProblemDefinition.tex

\section{Problem Definition}
This section presents a definition of the problem derived from the results of the problem analysis.
The essence of the problem is encapsulated in a problem statement at the end of the section, which the rest of the project will concern itself with developing a solution for.

The analysis results described in \autoref{maintenance} demonstrate the possible economical savings of using a predictive maintenance approach in manufacturing companies.

> This is caused by several benefits such as reduction in maintenance costs, increase in production and reduction of downtime.

Reviewer: "increase in production and reduction of downtime." These has the same meaning imo

Author: Det har ikke samme betydning. Forestil dig at en maskines effektivitet øges ved at man vedligeholder den på de rigtige tidspunkter, da slid kan sænke dens effektivitet. Slid behøver ikke betyde at maskinen bryder ned, men måske mindskes dens effektivitet og derved produktionen. Giver det mening?

Reviewer: Ja og nej. Jeg forstår dit argument, men i forhold til produktion er effektivitet baseret på on/off. Vedligeholdese burde ikke have effekt på hvor effektiv "on" er, men nærmere ratioen mellem on/off. Det giver lidt tvetydighed i terminologien, så vi skal nok lige tjekke op på definitionen. (Jeg er ikke sikker på om jeg har ret)

Reviewer: OEE er vist termet

Author: Så skal vi også lave en ændring i bunden af maintenance, for den der liste af praktiske økonomiske fordele ved predictive maintenance indeholder begge ting

Reviewer: "increase in production" er jo ikke en garanti, hvis det ikke referere til downtime. Det er en potentiel fordel alt efter maskinen. Minsket downtime kan derimod garanteres som fordel. 

But thats just my opinion man.

Author: Synes du at det er fint, det der står om det i slutningen af maintenance kapitlet?
For jeg kan sagtens fjerne "increase in production" her, men jeg vil gerne beholde det i maintenance :)

> This is caused by several benefits such as reduction in maintenance costs, increase in production and reduction of downtime.

Author: Se gerne i Thomas' review på linje 6, der taler vi om det.

> However, setting up the currently available solutions, as those described in \autoref{existingsolutions}, can be a time-consuming and non-trivial matter, making the decision of moving to a predictive approach less preferable. % Refactor this! What is the right argument? The source is ONLY neurospace; is that enough? Refer to previous section or provide source?

Author: Jeg ved ikke om det er den rigtige sætning du erstatter. Sætningen handlede om hvor besværligt det er at sætte op (ud fra neurospace's erfaring).
Du beskriver 2 ting:
- Den ene har jeg beskrevet i sætningen under, nemlig at man ikke er garanteret at det virker, før man har prøvet det.
- Den anden pointe du laver er noget helt andet, nemlig at man skal sørge for at få den rigtige konfiguration af løsningen. Hvad har det med problemet at gøre?

> However, setting up the currently available solutions, as those described in \autoref{existingsolutions}, can be a time-consuming and non-trivial matter, making the decision of moving to a predictive approach less preferable. % Refactor this! What is the right argument? The source is ONLY neurospace; is that enough? Refer to previous section or provide source?

Reviewer 2: Lige nu siger vi at det er besværligt at sætte op, men vi har ikke noget rigtigt belæg for dette udsagn. Så det skal erstattes med noget andet.
Vi kan jo ikke sige at SwiftSensors er svær at sætte op... Vi eller neurospace har jo testet det?
Kan vi undvære denne sætning?

Mit forsøg var en variation af det du nævner længere nede, så den behøver du ikke at tage med.

Companies could therefore benefit from discovering whether or not a predictive maintenance approach would result in economical savings, without having to spend large amounts of funding for a full predictive maintenance system.

A possible way to address this problem is to implement a temporary, incomprehensive system which measures data from the machines using sensors and serves the data to a data analysis component which attempts to predict maintenance for the machines.
Depending on the prediction results of this temporary solution, a company may decide to invest in a full-scale predictive maintenance system.
Multiple aspects of the problem such as sensor battery life-time and reliability are minimised in this manner, since the system would only have to be used for a limited time.
This makes it possible to provide a faster, less complicated and much cheaper solution for figuring out whether or not predictive maintenance is applicable for a given company.

> A solution like this should be flexible enough to support the maintenance of different types of machines in order to be applicable for different companies.

Author: Sætningen er til for at forklare betydningen af 'flexible'. Fleksibel betyder her at løsningen kan bruges i forskellige virksomheder. Er det vigtigt hvilke specifikke, tekniske ting det drejer sig om? På nuværende tidspunkt er det jo bare vigtigt at løsningen kan bruges i forskellige virksomheder.

> A solution like this should be flexible enough to support the maintenance of different types of machines in order to be applicable for different companies.

Reviewer: OEE er vist termet

> A solution like this should be flexible enough to support the maintenance of different types of machines in order to be applicable for different companies.

Reviewer: Tja, synes bare ikke forklaringen givet noget til forståelsen af ordet.

> A solution like this should be flexible enough to support the maintenance of different types of machines in order to be applicable for different companies.

Author: Det synes jeg. Kom med dine forslag.

> A solution like this should be flexible enough to support the maintenance of different types of machines in order to be applicable for different companies.

Reviewer: Det er sikkert fint for nu. Jeg har ikke et bedre forslag uden at snakke specifikke ting.

The solution should furthermore be desirable for a company to try out and therefore not be complicated and time-consuming to install.

Such a solution involves many aspects such as data collection, communication, storage, analysis e.g. machine learning, and a user interface for the company's maintenance personnel.
As mentioned in \autoref{introduction?}, this project focuses on the collection, communication and serving of the data from the manufacturing machines, while components such as data analysis and a front-end should be implemented for a comprehensive solution.

> The solution will be based on the problems and requirements provided by neurospace in regards to their experiences with predictive maintenance, and the aim is for neurospace to be able to use the solution by applying their data analysis technologies on the data collected. % Maybe add that the solution may of course be used by other parties?

Author: Jeg har prøvet at forbedre det. Check it out.

> The solution will be based on the problems and requirements provided by neurospace in regards to their experiences with predictive maintenance, and the aim is for neurospace to be able to use the solution by applying their data analysis technologies on the data collected. % Maybe add that the solution may of course be used by other parties?

Reviewer 2: ```suggestion
The solution will be based on the problems and requirements provided by neurospace in regards to their experiences with predictive maintenance, and the aim is for neurospace to determine if the manufacturing company can use predictive maintenance. % Maybe add that the solution may of course be used by other parties?
```

> The solution will be based on the problems and requirements provided by neurospace in regards to their experiences with predictive maintenance, and the aim is for neurospace to be able to use the solution by applying their data analysis technologies on the data collected. % Maybe add that the solution may of course be used by other parties?

Reviewer: Det nye er ganske godt

\subsection{Problem Statement}

The problem is expressed in the following problem statement:

> \center{\textit{How can a cheap, flexible and easy to install solution be developed to collect, store, and serve sensor data from manufacturing companies in order for neurospace to verify if predictive maintenance is applicable?}}

Reviewer: Burde det ikke være "production machines in manufacturing companies"? eller noget i den stil

Author: Jeg havde samme tanke, men... Det er svært at gøre en sætning perfekt, uden at den bliver træls at læse (*host* jura *host*), og jeg tror godt at læseren kan lægge 2 og 2 sammen, når vi taler om maintenance. Så er det selvfølgelig maskinerne vi taler om, dem har vi jo nævnt mange gange før.

Reviewer 2: Det er ikke helt tydeligt hvad vi mener med flexible ud fra dette afsnit.

Reviewer 2: Jo, linje 15 er nok tilstrækkelig. Dvs. med flexible, mener vi blot at systemet skal kunne bruges i forskellige produktionsvirksomheder? I så fald et det jo egentlig dækket at at vi skriver at den skal kunne: "collect, store, and serve sensor data from manufacturing companies"?
